textile/features.textile

```diff
@@ -1184,10 +1184,11 @@ h3(#activation-state-machine). Activation State Machine
 **** @(RSH3e1b)@ Transitions to @WaitingForRegistrationSync@.
 *** @(RSH3e2)@ On event @RegistrationSynced@:
 **** @(RSH3e2b)@ If the machine is in state @WaitingForRegistrationSync@ as a result of a @CalledActivate@ event, make @Push#activate@ return or call its callback with no error.
+**** @(RSH3e2c)@ Otherwise, calls the @updatedCallback@ provided to @Push#activate@ with no error.
 **** @(RSH3e2a)@ Transitions to @WaitingForNewPushDeviceDetails@.
 *** @(RSH3e3)@ On event @SyncRegistrationFailed@:
 **** @(RSH3e3c)@ If the machine is in state @WaitingForRegistrationSync@ as a result of a @CalledActivate@ event, make @Push#activate@ return or call its callback with the error.
-**** @(RSH3e3a)@ Otherwise, calls the @updateFailedCallback@ provided to @Push#activate@ with the error.
```

Comment:
Please could you take a look at the documentation about the [specification version](https://github.com/ably/specification#specification-version) and [modifying spec points](https://github.com/ably/specification/blob/main/CONTRIBUTING.md#features-spec-points)?

In particular, assuming that the intention is that we replace `updateFailedCallback` with `updatedCallback` (I don't know since you haven't updated the IDL) then the change proposed here is a breaking change — you're proposing changing the type of `updateFailedCallback` and also the circumstances in which it is called. If you want to introduce a breaking change, it requires a major version increase, and hence needs to go into the `integration/3.0.0` branch. But are you sure that you want to make a breaking change? Could we instead preserve (and deprecate) `updateFailedCallback` and add another optional `updatedCallback`? That, in fact, seems to be what you have done in [your ably-cocoa implementation of location pushes](https://github.com/ably/ably-cocoa/pull/1771/files#diff-007a192d0e318a7b962b4d5816211576f6f86bbdcf6f018373d587ead25d77e3).

Reply:
https://github.com/ably/specification/pull/160/commits/08294c01078b268cf8235af27addb8ffb038e7cc